packages/libs/api/sdk/package.json

```diff
@@ -12,15 +12,22 @@
   "types": "./src/index.d.ts",
   "dependencies": {
     "@apollo/client": "^3.6.9",
-    "graphql": "^16.5.0",
-    "cross-fetch": "^3.1.5"
+    "cross-fetch": "^3.1.5",
+    "graphql": "^16.5.0"
   },
   "devDependencies": {
+    "@graphql-codegen/cli": "2.13.8",
+    "@graphql-codegen/typed-document-node": "^2.3.5",
+    "@graphql-codegen/typescript": "2.8.0",
+    "@graphql-codegen/typescript-operations": "^2.5.5",
     "@pollyjs/adapter-node-http": "^6.0.5",
     "@pollyjs/core": "^6.0.5",
     "@pollyjs/persister-fs": "^6.0.5",
-    "@types/supertest": "^2.0.12",
     "@types/node": "16.11.7",
+    "@types/supertest": "^2.0.12",
     "supertest": "^6.2.4"
+  },
+  "scripts": {
+    "codegen": "graphql-codegen --config codegen.yml"
```

Comment:
per Nx logic might it be that this should be a separate project command instead @jkolyer @emanuelpinho?

Reply:
hmm, that's a good question. I don't know about nx best practices so it's better for @jkolyer to confirm indeed